src/main/java/edu/iss/powp/command/CommandDrawLineToPosition.java

```diff
@@ -0,0 +1,26 @@
+package edu.iss.powp.command;
+
+import edu.iis.client.plottermagic.IPlotter;
+
+public class CommandDrawLineToPosition implements PlotterCommand {
+
+	
+	private int x;
+	private int y;
+	
+	
+	
+	public CommandDrawLineToPosition(int x, int y) {
+		super();
+		this.x = x;
+		this.y = y;
+	}
+
+
+
+	@Override
+	public void execute(IPlotter iPlotter) {
```

Comment:
name plotter would be better than iPlotter. 'i' was supposed to indicate interface.